### PR TITLE
Add support for setting/querying a cluster-wide time zone

### PR DESCRIPTION
**Pull request by macintux**
Add interface to new riak core functionality (basho/riak_core#847 (RIAK-2639)).

### CODE REVIEW

**Comment by hazen** [comment]
@macintux Do we have the capability to view the current TZ setting?

**Comment by macintux** [comment]
Yes, `riak-admin timezone` with no additional argument will display the current setting.

File: rel/files/riak-admin [hunk 1]
```diff
@@ -1089,6 +1089,16 @@ case "$1" in
         node_up_check
         exec $NODETOOL rpc riak_core_console command $SCRIPT $@
         ;;
+    timezone)
+        if [ $# -gt 2 ]; then
+            echo "Usage: $SCRIPT $1"
+            echo "       $SCRIPT $1 <+-HHMM>"
```
Comment (hmmr) [inline rel/files/riak-admin]: The pedant in me suggests `<{+|-}HHMM>`.

**Comment by hmmr** [comment]
+1 a9b8c23

**Comment by macintux** [comment]
Should not be merged until basho/riak_core#847 (RIAK-2639) merges

**Comment by lukebakken** [comment]
Don't forget `riak_ee` ... :smile:

**Comment by macintux** [comment]
https://github.com/basho/riak_ee/pull/403

**Pull request merged**